Flutter/pdf-viewer/form-filling.md

```diff
@@ -668,6 +668,72 @@ Widget build(BuildContext context) {
 {% endhighlight %}
 {% endtabs %}
 
+## Undo, redo actions on Form fields
+
+If you performed any undesired actions when editing the form fields, you can undo and redo the action to restore the previous state. This section will go through how to undo and redo the changes made to the form fields.
```

Comment:
If you performed any undesired actions when editing the form fields, you can undo and redo the action to restore the previous state.

The undo and redo operations are performed by assigning the `UndoHistoryController` instance to the `undoController` property of the `SfPdfViewer`. The `UndoHistoryController` class contains the `undo` and 'redo` methods to perform the same, respectively. The `canUndo` and `canRedo` properties are used to check whether the undo and redo operations can be performed or not, respectively. The following code example illustrates how to perform the form filling undo and redo operations programmatically with the `SfPdfViewer`.

// code

For desktop platforms such as Windows, macOS, and desktop web, you can use the following shortcut keys to perform the actions:
 
Note: This `undoController` is common for both annotations and form fields. You do not need to create separate instances for both.